### ISSUE TITLE
Redot Logo needs to be desaturated for darker backgrounds

### ISSUE THREAD

**Issue by tokengamedev** [Redot-Engine/redot-engine#690]
### Tested versions

4.4

### System information

Windows 11

### Issue description

In case of darker backgrounds like in task bar or other places it looks odd and hurts the eye, but in white background it looks ok 
e.g., in windows startup compared to other red logos
![image](https://github.com/user-attachments/assets/5da0b04f-fe99-47fa-ab3a-b1494f118315)

the way to handle this without impacting hue is by reducing the saturation of colors.
currently the saturation is at 100%, it can be reduced by ~20%

The example given below is for comparison of various saturation values
![icons](https://github.com/user-attachments/assets/ee1c75d6-335f-4620-92c6-8e21d12fcd33)

If any other technique known, it can be applied too.

### Steps to reproduce

icons in editor

### Minimal reproduction project (MRP)

N.A

**Comment by heavymetalmixer** [comment]
Yeah, sometimes it's painful to look at it because of the saturation.

**Comment by Rabcor** [comment]
Yeah it's a bit of an eyesore on dark backgrounds, I agree.

**Comment by SkogiB** [comment]
This is a simple fix we can look at pushing in with the 4.3 stable PR we're working on rn

**Comment by tokengamedev** [comment]
just to add info, I have done this desaturation of image under sRGB color space (in Figma) since it is a Gradient.
I do not know how SVG is converted to Image under (meaning which color space). 
Ideally, I would like to have it in OKLab color space. I will check it out sometime later, but that will be a different issue/enhancement altogether

**Comment by SkogiB** [comment]
> just to add info, I have done this desaturation of image under sRGB color space (in Figma) since it is a Gradient. I do not know how SVG is converted to Image under (meaning which color space). Ideally, I would like to have it in OKLab color space. I will check it out sometime later, but that will be a different issue/enhancement altogether

The primary issue is that Godot's CI pipeline despises SVG changes and its a nightmare. We've been talking about improving that pipeline, but its a lower priority atm. We'll definitely keep an eye on this soon, so don't let us forget about this after we settle out the 4.3 rebrand. Hop in the dev chat sometime too, you do a lot of work in the background that's good.
